KKStream/playcraft-android-sdk
# chore(3509): Add androidTV playback sample module

## What

Add an androidTV sample.
Ticket link: https://kkvideo.atlassian.net/browse/OTP-3509

## Why

Let project side RD use Playcraft easier

## How

Add a new module that user can demo androidTV sample.

## Test Plan

Steps:
1. Run the tvsample module.
2. Verify if the functions operate normally or not.

## Checklist

Have you done all of these things?
